drivers/spi/spi_ll_stm32.c

```diff
@@ -307,12 +307,6 @@ static int spi_stm32_configure(struct device *dev,
 
 	LL_SPI_DisableCRC(spi);
 
```

Comment:
"This commit solves an issue where the NSS must be set before the
mode on the stm32mp157c_dk2, else LL_SPI_SetMode won't affect the
mode registry."

Do you mean that there is an h/w issue you are taking care of?
Does it make sense to mention some offiicial documentation about it?

Reply:
@avisconti 
No it is a SW bug. The SPI configuration sequence is not respected (please refer to stm32MP1 ref manual for details). If NSS is configured after the mode is set then a MODF error can occur.